config/routes.rb

```diff
@@ -460,6 +460,7 @@
         get ":slug(/:year)/proyecto/:id" => "plan_types#show", as: :project
         get ":slug(/:year)/tabla/:uid" => "plan_types#show"
         get ":slug(/:year)/tabla/:uid/:term_id" => "plan_types#show"
+        get ":slug(/:year)/dashboards(/:dashboard_id)" => "plan_types#show"
```

Comment:
This change should be included in the documentation in order for the module Dashboards to work, right?

Reply:
You may take a look https://gobierto.readme.io/docs/gobierto-dashboards

Reply:
👍🏾